packages/devtools/lib/src/timeline/frames_bar_chart.dart

```diff
@@ -67,10 +207,11 @@ class FrameBar extends CoreElement {
     layoutVertical();
 
     _initialize();
-
+/*
```

Comment:
Can FlameBar be deleted with the switch to Plotly?

Reply:
@kenzieschmoll can it it's a TODO I have in the code awaiting you response.  The only thing that seemed I should move over was _slowFrameWarning is that right?

Reply:
Yes that is correct. Keeping _slowFrameWarning would be good.